src/metrics/scenario.jl

```diff
@@ -180,3 +180,47 @@ end
 Calculate the mean coral evenness for each scenario for the entire domain.
 """
 scenario_evenness = Metric(_scenario_evenness, (:timesteps, :scenarios))
+
+"""
+    scenario_outcomes(rs::ResultSet, metrics::Vector{Metric})::NamedDimsArray
+
+Get outcomes for a given list of metrics and a result set
+
+# Arguments
+    - `rs` : ResultSet
+    - `metrics` : Vector of scenario Metrics (the ones that start "with scenario_")
+
+# Returns
+    NamedDimsArray with (:timesteps, :scenarios, :outcomes)
+
+# Examples
+```
+metrics::Vector{ADRIA.metrics.Metric} = [
+    ADRIA.metrics.scenario_total_cover,
+    ADRIA.metrics.scenario_asv,
+    ADRIA.metrics.scenario_absolute_juveniles,
+]
+
+# 3-dimensional Array of outcomes
+outcomes = ADRIA.metrics.scenario_outcomes(rs, metrics)
+```
+"""
+function scenario_outcomes(rs::ResultSet, metrics::Vector{Metric})::NamedDimsArray
+    num_timesteps = length(timesteps(rs))
+    num_scenarios = size(rs.inputs, 1)
+    num_metrics = length(metrics)
+
+    outcomes = NamedDimsArray(
+        zeros(Float64, num_timesteps, num_scenarios, num_metrics);
+        timesteps=timesteps(rs),
+        scenarios=1:num_scenarios,
+        outcomes=name_symbol.(metrics),
+    )
+
+    for (i, metric) in enumerate(metrics)
+        outcome = metric(rs)
+        outcomes[:, :, i] = outcome
```

Comment:
I think this is overly verbose for not much gain in readability.


Suggest:

```suggestion
        outcomes[:, :, i] = metric(rs)
```

"Assign result of metric to outcomes", 

as opposed to

"metric to outcome, then outcome to outcomes"